tair/taircommands.go

```diff
@@ -12,13 +13,13 @@ type tairCmdable func(ctx context.Context, cmd redis.Cmder) error
 type TairCmdable interface {
 	// TairZset
 	exZAdd(ctx context.Context, key string, p *ExZAddArgs, member ...ExZAddMember) *redis.IntCmd
-	ExZAddManyScore(ctx context.Context, key string, member string, scores ...float64) *redis.IntCmd
```

Comment:
float is right api，we need change `joinScoresToString` function to float.
refer [here](https://github.com/alibaba/alibabacloud-tairjedis-sdk/blob/master/src/main/java/com/aliyun/tair/tairzset/LeaderBoard.java#L66)

Reply:
pr it